tarantool_test.go

```diff
@@ -3126,6 +3126,7 @@ func TestConnectionFeatureOptsImmutable(t *testing.T) {
 
 	restartOpts := startOpts
 	restartOpts.Listen = "127.0.0.1:3014"
+	restartOpts.WorkDir = "work_dir2"
```

Comment:
I think it would be great to use temporary directories instead of `work_dirX` in the local folder.
https://pkg.go.dev/io/ioutil#TempDir

Reply:
> Deprecated

Reply:
But it seems it's possible with https://pkg.go.dev/os#MkdirTemp

Reply:
> > Deprecated

Since 1.17, but we supports go 1.13+, so we can not use os.MkdirTemp (since 1.16).

Reply:
Added, fixed some issues

Reply:
Close in favor of comments below